luminoso_api/v5_download.py

```diff
@@ -91,7 +92,7 @@ def download_docs(client, output_filename=None, expanded=False):
             print(json.dumps(doc, ensure_ascii=False), file=out)
 
 
-def main():
+def _main(argv):
```

Comment:
This was originally just `main`, rather than passing the arguments in explicitly from a different `main`.  For the sake of having diffs that make sense, I think it should be returned to that structure.

Reply:
It seems a lot harder to test this (in lumi-stack-tools) without the ability to pass in arguments.